packages/xfce4_terminal.rb

```diff
@@ -36,5 +37,13 @@ def self.build
 
   def self.install
     system "make install DESTDIR=#{CREW_DEST_DIR}"
+    system "cat <<'EOF'> xfce4-terminal
+WAYLAND_DISPLAY=wayland-0
```

Comment:
Probably should add `#!/bin/bash` at the top.

Reply:
Done